modules/ROOT/pages/penetration-testing-policies.adoc

```diff
@@ -9,40 +9,78 @@ image:logo-hybrid-disabled.png[link="/runtime-manager/deployment-strategies#hybr
+To request a Security Assessment:
 
-* Start and end time of testing.
-* Hosts to include in the test.
-* Source IP address or range.
-* Contact information, including a phone number and email, for the individual or team conducting the test.
-* You agree to only audit CloudHub workers, and not audit any other platform services which are common to all tenants.
-* You agree not to audit shared workers.
-* Send an email to security@mulesoft.com upon start and completion of tests.
-* Notify security@mulesoft.com of any vulnerabilities found immediately.
-* You may only audit once a quarter.
+. Contact your Salesforce Account team to request the Security Assessment Request form and Security Assessment Agreement.
+. Complete the request form and accept the agreement.
++
+For information about how to complete the request form, see the knowledge article https://help.salesforce.com/articleView?id=000322336&type=1&mode=1[Complete the Security Assessment submission form].
+. Send your completed Security Assessment form to mailto:securityassessment@salesforce.com[securityassessment@salesforce.com].
++
+Your account representative contacts you when your Security Assessment request is approved.
 
-After you submit your request, you receive a written authorization from the MuleSoft team, at which point you can proceed with testing according to your proposed plan.
+== Run Penetration and Vulnerability Tests  
+
+Because penetration testing could interfere with other tenants, MuleSoft allows penetration testing 
+on your workers but not other Anypoint Platform services.
+
+=== Prerequisites
+
+Before initiating penetration testing, enable static IPs for the app.
+
+By default, CloudHub workers do not use static IP addresses, so you can't test them because their IP addresses might change.
+For information about how to enable static IPs, see xref:deploying-to-cloudhub.adoc#static-ips-tab-settings[Static IPs Tab Settings].
+
+
+=== Steps 
+
+After your Security Assessment request is approved, follow these steps:
+
+. Run the penetration tests.
++
+The Security Assessment Agreement includes restrictions and requirements for testing.
+. Assign a security resource at your company to review and validate findings from the tests.
++
+[IMPORTANT]
+You must complete this step before following up with Salesforce. 
++
+The following documents help the reviewer identify common false positives or security issues related to settings:
++
+--
+** https://help.salesforce.com/articleView?id=Salesforce-Platform-Security-FAQs&type=1&language=en_US[Platform Security FAQ]
+** https://help.salesforce.com[Salesforce Help]
+--
+
+. Send any outstanding security vulnerability findings to mailto:security@salesforce.com[security@salesforce.com].
++
+--
+For information about how to submit a security vulnerability finding, see the 
+https://help.salesforce.com/articleView?id=000320207&type=1&mode=1[Security Vulnerability Finding Submittal Guide].
+
```

Comment:
So, I'm feeling a little uncomfortable about how much this task relies on some smaller subtasks, like having the resource person check the results, and the resources required for that, and details about the information that needs submitting in the email to Security. 
Also the need for the notices, which COULD be plain text but I guess not if these are common problems with users....